oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/server/ServerBearerTokenAuthenticationConverter.java

```diff
@@ -90,8 +91,20 @@ public void setAllowUriQueryParameter(boolean allowUriQueryParameter) {
 		this.allowUriQueryParameter = allowUriQueryParameter;
 	}
 
-	private static String resolveFromAuthorizationHeader(HttpHeaders headers) {
-		String authorization = headers.getFirst(HttpHeaders.AUTHORIZATION);
+	/**
+	 * Set this value to configure what header is checked when resolving a Bearer Token.
+	 * This value is defaulted to {@link HttpHeaders#AUTHORIZATION}.
+	 *
+	 * This allows other headers to be used such as {@link HttpHeaders#PROXY_AUTHORIZATION}
+	 *
+	 * @param authorizationHeader the header to check when retrieving the Bearer Token.
```

Comment:
Since this will go out in 5.4, will you please add a `@since 5.4` to the JavaDoc?